microsoft-edge/webview2/release-notes/index.md

```diff
@@ -0,0 +1,1318 @@
+---
+title: Release Notes for the WebView2 SDK
+description: Release notes for Microsoft Edge WebView2, covering new features, APIs, and fixes for Win32, WPF, and WinForms.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 03/18/2024
+---
+# Release Notes for the WebView2 SDK
+
+These Release Notes provide information about new features and non-security updates that are included in the Microsoft Edge Stable Channel.
+<!-- todo: 
+is that true for the WebView2 Prerelease SDK, or only for the WebView2 Release SDK?
+mention "WebView2", "Runtime", & "SDK"?
```

Comment:
In the above sentence, mention "WebView2", "Runtime", & "SDK"?
@victorhuangwq

Reply:
These Release Notes provide information about new features and bug fixes that are included in the WebView2 Release SDK.

Reply:
but "Release" is wrong, Prerelease also